coveo-settings/coveo_settings/settings.py

```diff
@@ -172,7 +174,8 @@ def _raise_if_missing(self) -> None:
             raise MandatoryConfigurationError(f'Mandatory config item "{self.key}" is missing.')
 
     def _pretty_repr(self, value: Optional[ConfigValue]) -> str:
-        return f'{self.__class__.__name__}[{self.key}] = {"<not-set>" if value is None else value}'
+        value_str = "<not-set>" if value is None else "<sensitive>" if self._sensitive else value
```

Comment:
I never really liked nested ternary operator, I find it hard to read but it's personal preference.

Reply:
the 3.10 update will make these much nicer with the match/case syntax.